<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-9AR9G70" failures="1" tests="1" name="Testng(failed)(failed)(failed)" time="0.094" errors="0" timestamp="2022-02-25T11:11:39 IST">
  <testcase classname="com.aspire.TestNg.FailedFiles.FailedFile" name="TCS2" time="0.016">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.aspire.TestNg.FailedFiles.FailedFile.TCS2(FailedFile.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TCS2 -->
  <testcase classname="com.aspire.TestNg.FailedFiles.FailedFile" name="TCS5" time="0.0">
    <ignored/>
  </testcase> <!-- TCS5 -->
  <testcase classname="com.aspire.TestNg.FailedFiles.FailedFile" name="TCS1" time="0.0">
    <ignored/>
  </testcase> <!-- TCS1 -->
  <testcase classname="com.aspire.TestNg.FailedFiles.FailedFile" name="TCS4" time="0.0">
    <ignored/>
  </testcase> <!-- TCS4 -->
  <testcase classname="com.aspire.TestNg.FailedFiles.FailedFile" name="TCS3" time="0.0">
    <ignored/>
  </testcase> <!-- TCS3 -->
</testsuite> <!-- Testng(failed)(failed)(failed) -->
